packages/marko-web-identity-x/config.js

```diff
@@ -6,16 +6,27 @@ const validHooks = ['onAuthenticationSuccess'];
 class IdentityXConfiguration {
   /**
    *
-   * @param {object|string} options When a string, assumes an `appId`, else options object.
-   * @param {array} [options.requiredServerFields] Required fields, server enforced.
-   * @param {array} [options.requiredClientFields] Required fields, client-side only.
+   * @param {object} options
+   * @param {string} options.appId The application ID to use.
+   * @param {string} [options.apiToken] An API token to use. Only required when doing write ops.
+   * @param {string[]} [options.requiredServerFields] Required fields, server enforced.
+   * @param {string[]} [options.requiredClientFields] Required fields, client-side only.
    */
-  constructor(options) {
-    // BC check for when the constructor only had a single `appId` argument.
```

Comment:
Note, the BC check is no longer needed within the parameter1 repo, hence why it was removed.